src/app/modules/main/profile_section/wallet/networks/item.nim

```diff
@@ -125,6 +133,20 @@ QtObject:
     read = fallbackURL
     notify = fallbackURLChanged
 
+  proc originalRpcURLChanged*(self: Item) {.signal.}
+  proc originalRpcURL*(self: Item): string {.slot.} =
+    return self.originalRpcURL
+  QtProperty[string] originalRpcURL:
```

Comment:
I don't think you need the QrProperties in this file. Since Items are used in Models, it's the model that notifies the UI that a property has changed with `dataChanged` or `reset`.

Unless the UI will even have access to a single Item on its own, we will never need slots and signals. 

It,s just an FYI, as it's not a big deal, but it would simplify the code.

Reply:
its used in 2 places once exposed via the network/model.nim and other via the combined_item.nim and in this case as a QTObject and using its properties

Reply:
The items are qt object and exposed hence why it works ;)